src/ftest.jl

```diff
@@ -0,0 +1,130 @@
+type FTestTable
+    ntests::Int
```

Comment:
True, but I was thinking that length() on the vectors was cumbersome. I'd replace this with a function ntests(t::FTestTable), but there's no point if we're going to CoefTable